### PR TITLE
Invert resource loading feature flag logic

### PR DESCRIPTION
**Pull request by kevinzheng-ap**
Invert new resource loading feature flag to be enabled by default.

cc @jrodbx

### CODE REVIEW

File: paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt [hunk 1]
```diff
@@ -301,14 +301,23 @@ class PaparazziPluginTest {
   }
 
   @Test
-  fun flagNewResourceLoadingIsOn() {
-    val fixtureRoot = File("src/test/projects/flag-new-resource-loading-on")
+  fun flagOldResourceLoadingIsOff() {
+    val fixtureRoot = File("src/test/projects/flag-old-resource-loading-off")
```
Comment (jrodbx) [inline paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt:305]:
```suggestion
  fun flagLegacyResourceLoadingIsOff() {
    val fixtureRoot = File("src/test/projects/flag-legacy-resource-loading-off")
```

File: paparazzi-gradle-plugin/src/test/projects/flag-old-resource-loading-on/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+android.useAndroidX=true
+app.cash.paparazzi.old.resource.loading=true
```
Comment (jrodbx) [inline paparazzi-gradle-plugin/src/test/projects/flag-old-resource-loading-on/gradle.properties:2]:
```suggestion
app.cash.paparazzi.legacy.resource.loading=true
```

File: paparazzi-gradle-plugin/src/test/projects/invalid-application-plugin/gradle.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+android.useAndroidX=true
+app.cash.paparazzi.old.resource.loading=true
```
Comment (jrodbx) [inline paparazzi-gradle-plugin/src/test/projects/invalid-application-plugin/gradle.properties:2]:
```suggestion
app.cash.paparazzi.legacy.resource.loading=true
```

File: paparazzi-gradle-plugin/src/test/projects/missing-library-plugin/gradle.properties [hunk 4]
```diff
@@ -0,0 +1,2 @@
+android.useAndroidX=true
+app.cash.paparazzi.old.resource.loading=true
```
Comment (jrodbx) [inline paparazzi-gradle-plugin/src/test/projects/missing-library-plugin/gradle.properties:2]:
```suggestion
app.cash.paparazzi.legacy.resource.loading=true
```

File: paparazzi/src/main/java/app/cash/paparazzi/internal/Renderer.kt [hunk 5]
```diff
@@ -51,9 +51,9 @@ internal class Renderer(
   fun prepare(): SessionParamsBuilder {
     val platformDataResDir = File("${environment.platformDir}/data/res")
 
-    val useNewResourceLoading = System.getProperty(Flags.NEW_RESOURCE_LOADING).toBoolean()
+    val useOldResourceLoading = System.getProperty(Flags.OLD_RESOURCE_LOADING).toBoolean()
```
Comment (jrodbx) [inline paparazzi/src/main/java/app/cash/paparazzi/internal/Renderer.kt:54]:
```suggestion
    val useLegacyResourceLoading = System.getProperty(Flags.LEGACY_RESOURCE_LOADING).toBoolean()
```

File: paparazzi/src/main/java/app/cash/paparazzi/internal/Renderer.kt [hunk 6]
```diff
@@ -51,9 +51,9 @@ internal class Renderer(
...
     val (frameworkResources, projectResources) =
-      if (!useNewResourceLoading) {
+      if (useOldResourceLoading) {
```
Comment (jrodbx) [inline paparazzi/src/main/java/app/cash/paparazzi/internal/Renderer.kt:56]:
```suggestion
      if (useLegacyResourceLoading) {
```

File: paparazzi/src/main/java/app/cash/paparazzi/Flags.kt [hunk 7]
```diff
@@ -2,5 +2,5 @@ package app.cash.paparazzi
 
 object Flags {
   const val DEBUG_LINKED_OBJECTS = "app.cash.paparazzi.debug.linked.objects"
-  const val NEW_RESOURCE_LOADING = "app.cash.paparazzi.new.resource.loading"
+  const val OLD_RESOURCE_LOADING = "app.cash.paparazzi.old.resource.loading"
```
Comment (jrodbx) [inline paparazzi/src/main/java/app/cash/paparazzi/Flags.kt:5]:
```suggestion
  const val LEGACY_RESOURCE_LOADING = "app.cash.paparazzi.legacy.resource.loading"
```

**Comment by kevinzheng-ap** [comment]
@jrodbx Could you have this PR merged? Thanks

**Pull request merged**